backend-api/template.yaml

```diff
@@ -1155,6 +1155,126 @@ Resources:
         - !Ref PermissionsBoundary
         - !Ref AWS::NoValue
 
+  WellKnown5XXLowThresholdAlarm:    
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmDescription: "/.well-known/jwks.json endpoint has returned at least 2 5XX status codes twice in the past 5 minutes"
```

Comment:
It's not entirely clear from the ticket, but I think we should align this to the High Threshold alarm:

1% or more of requests are failing on the /.well-known/jwks.json endpoint